package.json

```diff
@@ -18,6 +18,7 @@
   },
   "files": ["src", "dist"],
   "jest": {
+    "testURL": "http://localhost/",
```

Comment:
This is needed by the new version of JSDom that I got. See https://github.com/jsdom/jsdom/issues/2304 for details